build-system/server/lazy-build.js

```diff
@@ -134,7 +134,6 @@ async function preBuildRuntimeFiles() {
   await build(jsBundles, 'amp.js', (_bundles, _name, options) =>
     compileCoreRuntime(options)
   );
-  await build(jsBundles, 'ww.max.js', doBuildJs);
```

Comment:
We had to pre-build `ww.max.js` because test files didn't request it by its correct name. Is this no longer the case? 

See https://github.com/ampproject/amphtml/issues/24226#issue-485568433 for context.